pkg/flowkit/state_test.go

```diff
@@ -526,7 +643,7 @@ func Test_ChangingState(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, (*pkey).String(), pk.String())
 
-	bar, err := p.Accounts().ByAddress(flow.HexToAddress("0x1"))
+	bar, err := p.Accounts().ByAddressAndAccountName(flow.HexToAddress("0x1"), "foo")
```

Comment:
will need to delete existing byaddress tests since it is no longer used, if this PR is merged.